README.md

```diff
@@ -89,6 +89,8 @@ Please refer to the details on how the ngx-prefetch works [here](docs/HOW_IT_WOR
   If this is not the case, you can configure the full path of the resources that will be prefetched (ex: https://my-web-app.com/path/to/my-app/). 
   It is also possible to set this value at runtime. Instead of setting it in the Builder's options, you can search for `{STATICS_FULL_PATH}` and replace it on the server side in order to inject a path.
 
+  - `localizationPattern` Pattern for the relative path of the localization file. By default, the pattern corresponds to the JSON file in a folder called localizations: `"/localizations/${language}.json"`.
```

Comment:
we should mention that if path contains the ${language} part, it will be replaced on the server side.